tools/check/ut.go

```diff
@@ -902,7 +901,7 @@ func generateBuildCache() error {
 	cmd := exec.Command("go", "test", "-tags=intest", "-exec=true", "-vet=off")
 	goCompileWithoutLink := fmt.Sprintf("-toolexec=%s/tools/check/go-compile-without-link.sh", workDir)
 	cmd.Args = append(cmd.Args, goCompileWithoutLink)
-	cmd.Dir = path.Join(workDir, "cmd/tidb-server")
+	cmd.Dir = filepath.Join(workDir, "cmd/tidb-server")
```

Comment:
If we consider different operating systems, dose `cmd/tidb-server` also need to be changed?